devices/src/virtio/block.rs

```diff
@@ -40,8 +40,6 @@ const QUEUE_SIZES: &[u16] = &[QUEUE_SIZE];
 const QUEUE_AVAIL_EVENT: DeviceEventT = 0;
 // Rate limiter budget is now available.
 const RATE_LIMITER_EVENT: DeviceEventT = 1;
-// Backing file on the host has changed.
-pub const FS_UPDATE_EVENT: DeviceEventT = 2;
 // Number of DeviceEventT events supported by this implementation.
 pub const BLOCK_EVENTS_COUNT: usize = 3;
```

Comment:
The count changes because you remove an event.

```suggestion
pub const BLOCK_EVENTS_COUNT: usize = 2;
```

Reply:
Nice catch. Done.